src/lro/constants.ts

```diff
@@ -0,0 +1 @@
+export const terminalStates = ["succeeded", "failed", "canceled", "cancelled"];
```

Comment:
That's kind of interesting that we support both spellings for the past tense of `cancel` :smile:

Reply:
I included it since I saw it in the v5 version. I thought it wouldn't hurt to have 😄